tests/Common/IndexTest.php

```diff
@@ -63,11 +63,14 @@ public function testIndexExclude(): void
         $this->assertArrayHasKey(0, $componentsWithoutDetails);
 
         $firstComponent = $componentsWithoutDetails[0];
-        $this->assertCount(4, $firstComponent);
+        $this->assertCount(7, $firstComponent);
```

Comment:
Este som opravil ten count tag na tom spadol

Reply:
Aha, to som si nevsimol. Vyhodil by som ten count uplne najradsej. Ak sa testuje existencia kazdeho fieldu, nemusime ich zratat.

Reply:
Jo kludne to vyhod

Reply:
@fejjo vyhodene. Super, ze si si to vsimol.